x/interchainstaking/keeper/ibc_packet_handlers_test.go

```diff
@@ -2862,6 +2863,78 @@ func (suite *KeeperTestSuite) TestGetValidatorForToken() {
 	}
 }
 
+func (suite *KeeperTestSuite) TestHandleCompleteSend() {
+	suite.Run("TestHandleCompleteSend", func() {
```

Comment:
Hey plz move this setup after line 2917 so we can know the name of each test case when we can run the testing